orangecontrib/imageanalytics/widgets/tests/test_owsaveimages.py

```diff
@@ -137,7 +130,7 @@ def test_save_file_calls_save_as(self):
         widget.save_file_as.assert_called()
         widget.save_file_as.reset_mock()
 
-        widget.dirname = "bar"
+        widget.dirname = os.path.join(self.test_dir, "bar")
```

Comment:
This test saved images in the directory from where tests are running. It is better to save in temp dir, which is later removed.